pkg/util/fs_util_test.go

```diff
@@ -694,3 +694,49 @@ func Test_childDirInWhitelist(t *testing.T) {
 		})
 	}
 }
+
+func Test_correctDockerignoreFileIsUsed(t *testing.T) {
+	type args struct {
+		dockerfilepath string
+		buildcontext   string
+		excluded       string
+		notExcluded    string
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		{
+			name: "relative dockerfile used",
+			args: args{
+				dockerfilepath: "../../integration/dockerfiles/Dockerfile_test_dockerignore_relative",
+				buildcontext:   "../../integration/",
+				excluded:       "ignore_relative/bar",
+				notExcluded:    "ignore_relative/foo",
```

Comment:
i think we should test `ignore/bar` is not excluded along with `ignore_relative/foo` here to make sure context dockerignore is not used.? What do you think?

Reply:
very good point, yes!